python/MOCK_AIS.py

```diff
@@ -111,46 +134,85 @@ def set_random_seed(self):
             random.seed(randomSeed)
             rospy.loginfo("randomSeed = {}. Setting seed".format(randomSeed))
         except ValueError:
-            rospy.logwarn("randomSeed = {}. Not setting seed".format(randomSeed))
+            rospy.logwarn(
+                "randomSeed = {}. Not setting seed".format(randomSeed))
 
     def move_ships(self):
         speedup = rospy.get_param('speedup', default=1.0)
         movement_time_seconds = speedup * self.publishPeriodSeconds
 
-        for i in range(self.numShips):
-            self.ships[i].move(movement_time_seconds)
+        
+        for i in (i for i in range(len(self.ships)) if self.ships):
```

Comment:
Is this line too verbose? I thought it would be cleaner to use a generator expression instead of this, both are ok to me:
`
for i in range(len(self.ships))
    if self.ships
       #rest of code here


`